Mimir.MongoDB/Bson/Serialization/Serializers/Lib9c/Items/EquipmentSerializer.cs

```diff
@@ -0,0 +1,93 @@
+using Lib9c.Models.Items;
+using Mimir.MongoDB.Bson.Extensions;
+using Mimir.MongoDB.Bson.Serialization.Serializers.Lib9c.Skills;
+using Mimir.MongoDB.Bson.Serialization.Serializers.Lib9c.Stats;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Serializers;
+
+namespace Mimir.MongoDB.Bson.Serialization.Serializers.Lib9c.Items;
+
+public class EquipmentSerializer : ClassSerializerBase<Equipment>
+{
+    public static readonly EquipmentSerializer Instance = new();
+
+    public static Equipment Deserialize(BsonDocument doc)
+    {
+        if (!doc.TryGetValue("ItemType", out var itemTypeValue))
+        {
+            throw new BsonSerializationException("Missing ItemType in document.");
+        }
+
+        if (!doc.TryGetValue("ItemSubType", out var itemSubTypeValue))
+        {
+            throw new BsonSerializationException("Missing itemSubTypeValue in document.");
+        }
+
+        var itemType = (Nekoyume.Model.Item.ItemType)itemTypeValue.AsInt32;
+        var itemSubType = (Nekoyume.Model.Item.ItemSubType)itemSubTypeValue.AsInt32;
```

Comment:
It is just a note, we should store as string because the `Nekoyume.Model.Item.ItemType` doesn't specify number for each item. (e.g., `Consumable = 1`)

Reply:
Thanks to aware that.